packages/platform/platform-fastify/src/components/PlatformFastify.ts

```
@@ -0,0 +1,337 @@
+    const env = constant<Env>("env");
+
+    const promises = plugins.map(async (plugin: PlatformFastifyPluginSettings): Promise<PlatformFastifyPluginLoadingOptions> => {
+      if (isFunction(plugin)) {
+        return {
+          env,
+          use: plugin
+        };
+      }
+
+      if (isString(plugin)) {
+        plugin = {env, use: plugin};
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)